### PR TITLE
Fix bug in workoutsessionAdd parser, fixed some ui for workout and di…

### PR DESCRIPTION
**Pull request by yujinyang1998**
…et, updated UG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -40,7 +40,8 @@ By: `Team F11-1` Since: `Sept 2020` License: `MIT`
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3.4.2.2. [Adding a Move to the Current Workout Session: `add`](#ws-add)<br>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3.4.2.3. [Listing All Moves from the Current Workout Session: `list`](#ws-list)<br>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3.4.2.4. [Deleting a Move From the Current Workout Session: `delete`](#ws-delete)<br>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3.4.2.5. [Ending the Current Workout Session: `end`](#ws-end)<br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3.4.2.5. [Deleting a Move From the Current Workout Session: `search`](#ws-search)<br>
```
Comment (tienkhoa16) [inline docs/UserGuide.md]: Should the title be "Searching for a Move from the Current Workout Session"?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -766,7 +767,27 @@ Expected outcome:
      _________________________________________________________________________________________________
 ```
 
-#### 3.4.2.5. <a id="ws-end">Ending the Current Workout Session: `end`</a>
+
+#### 3.4.2.5. <a id="ws-end">Searching for a keyword in the Current Workout Session: `search`</a>
```
Comment (tienkhoa16) [inline docs/UserGuide.md]: Change this title to sync with the title in table of contents

Comment (tienkhoa16) [reply docs/UserGuide.md]: and change the anchor to `<a id="ws-search">`

Comment (yujinyang1998) [reply docs/UserGuide.md]: great catch!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -766,7 +767,27 @@ Expected outcome:
      _________________________________________________________________________________________________
 ```
 
-#### 3.4.2.5. <a id="ws-end">Ending the Current Workout Session: `end`</a>
+
+#### 3.4.2.5. <a id="ws-search">Searching for a keyword in the Current Workout Session: `search`</a>
+Ends the current workout session and saves the relevant data.
```
Comment (tienkhoa16) [inline docs/UserGuide.md]: The feature description does not match

Comment (yujinyang1998) [reply docs/UserGuide.md]: damn I missed alot of stuff lol

**Review by tienkhoa16 — APPROVED** [review]
It's good to goooooo

**Pull request merged**